src/Tizen.NUI/src/internal/Application/Application.cs

```diff
@@ -684,12 +684,6 @@ private void OnApplicationInit(IntPtr data)
             DisposeQueue.Instance.Initialize();
             Tizen.Tracer.End();
 
-            Log.Info("NUI", "[NUI] OnApplicationInit: ProcessorController Initialize");
-            Tizen.Tracer.Begin("[NUI] OnApplicationInit: ProcessorController Initialize");
-            // Initialize ProcessorController Singleton class. This is also required to create ProcessorController on main thread.
-            ProcessorController.Instance.Initialize();
```

Comment:
Is it ok if ProcessorController is not initialized here?

Reply:
Yes. We don't need to initialize it now. Previously this API was exist due to we need to assume that ProcessController callback registered at main thread.

But mostly it was unneccessary worries, and if someone create it before application initialized, applicaton would be crashed, since API9. So I think we can remove it.